.github/workflows/release.yml

```diff
@@ -15,18 +15,20 @@ on:
           - prod
       APPS:
         type: string
-        description: "The apps to release with this format: `app-dir1=version1,app-dir2=version2`. E. g.: `go-highs-knapsack=v1.2.0,java-ortools-knapsack=v1.2.0`"
+        description: "The apps to release, comma separated. E. g.: `go-highs-knapsack,java-ortools-knapsack`"
```

Comment:
Will the manual part of this workflow stay? When releasing manually, we should probably not bump the version, right? 
I.e., if we run against dev/staging, we don't want to mess with the version of the app in the repo, but rather just push it there, right? :thinking:

Reply:
I left the manual dispatch in case we needed to release by hand (although we shouldn't need to). In any case, the inputs are just names, the version is not specified anymore, in neither the manual nor the automated way.